src/pint/scripts/event_optimize.py

```diff
@@ -545,6 +621,12 @@ def main(argv=None):
         default=False,
         action="store_true",
     )
+    parser.add_argument(
+        "--autocorr",
+        help="Turn the autocorrelation check function off",
```

Comment:
What is the behavior of event_optimize when "--autocorr" is given? If it is to turn off the autocorrelation, I think this option will be better called "--no-autocorr".

Reply:
I've updated the option now